### PR TITLE
Upgrade Code-Aster Adapter for preCICE 2.0

### LINKED ISSUE #15: Adapter needs to be ported to preCICE v2
See the [migration guide](https://github.com/precice/precice/wiki/Porting-adapters-from-preCICE-1.x-to-2.x).

### PR DESCRIPTION
**Pull request by micheltakken**
Solve conflicts from PR #17 

Closes #15.

### CODE REVIEW

File: cht/adapter.py [hunk 1]
```diff
@@ -21,8 +21,8 @@ def __init__(self, preciceConfigFile, participantName, config, MESH, MODEL, MAT,
 		self.isNonLinear = isNonLinear
 		self.participantName = participantName
 		self.preciceDt = -1
-		self.precice = precice.Interface(participantName, 0, 1)
-		self.precice.configure(preciceConfigFile)
+		self.precice = precice.Interface(participantName, preciceConfigFile, 0, 1)
+		# self.precice.configure(preciceConfigFile)
```
Comment (MakisH) [inline cht/adapter.py]:
```suggestion
```

File: tutorials/buoyantSimpleFoam-aster/precice-config.xml [hunk 2]
```diff
@@ -31,6 +31,7 @@
       <read-data mesh="Solid-Interface-Faces" name="Sink-Temperature-Fluid"/>
       <read-data mesh="Solid-Interface-Faces" name="Heat-Transfer-Coefficient-Fluid"/>
       <mapping:nearest-neighbor constraint="consistent" direction="read" to="Solid-Interface-Faces" from="Fluid-Interface"/>
+       <export:vtk directory="preCICE-output" />
```
Comment (MakisH) [inline tutorials/buoyantSimpleFoam-aster/precice-config.xml]: This is an additional setting, not related to preCICE v2. I would suggest that we remove it, for consistency with other tutorials.

```suggestion
```

File: tutorials/buoyantSimpleFoam-aster/precice-config.xml [hunk 3]
```diff
@@ -40,11 +41,12 @@
       <read-data mesh="Fluid-Interface" name="Sink-Temperature-Solid"/>
       <read-data mesh="Fluid-Interface" name="Heat-Transfer-Coefficient-Solid"/>
       <mapping:nearest-neighbor constraint="consistent" direction="read" to="Fluid-Interface" from="Solid-Interface-Nodes"/>
+      <export:vtk directory="preCICE-output" />
```
Comment (MakisH) [inline tutorials/buoyantSimpleFoam-aster/precice-config.xml]: Same here:
```suggestion
```

File: cht/adapter.py (hunk 1 above)
Comment (micheltakken) [reply cht/adapter.py]: Solved

File: tutorials/buoyantSimpleFoam-aster/precice-config.xml (hunk 2 above)
Comment (micheltakken) [reply tutorials/buoyantSimpleFoam-aster/precice-config.xml]: Solved

File: tutorials/buoyantSimpleFoam-aster/precice-config.xml (hunk 3 above)
Comment (micheltakken) [reply tutorials/buoyantSimpleFoam-aster/precice-config.xml]: Solved

**Pull request merged**